CHANGELOG.md

```diff
@@ -122,6 +132,19 @@ These should not affect the functionality of the module.
 
 - dns\_lookup.rb: support arrays of names [\#1](https://github.com/voxpupuli/puppet-dnsquery/pull/1) ([pcarrier](https://github.com/pcarrier))
 
+3.0.0:
+ * Add an optional lambda argument that is called and returned instead if
+   the lookup raises an exception or returns an empty result.
+ * Change return format on dns_srv and dns_mx to an array of hashes. This
+   removes the need for the optional field parameter. That can be fetched using
+   the map function in Puppet 4.x+ instead.
+ * Use the Puppet 4 function API.
+
+2.0.1:
+ * correctly convert array of Resolv::IPv4 or Resolv::IPv6 to array strings
+
+2.0.0:
+ * Raise an exception on empty replies.
```

Comment:
Where does this come from?

Reply:
automatically generated by the rake task, but i dont know why it either wasnt there previously or is there now

Reply:
https://github.com/voxpupuli/puppet-dnsquery/blob/master/HISTORY.md

Reply:
ahaaaa, will do another PR to fix that and rebase this one then